.ci/aarch64/lib_install_qemu_aarch64.sh

```diff
@@ -66,6 +66,12 @@ build_and_install_qemu() {
                 patch -p1 <"$patch"
         done
 
+	# Replace the default-configs/*-softmmu.mak with the tuned ones
+        echo "Overwriting default qemu kconfigs"
+        if [ -e ${GOPATH}/src/${PACKAGING_REPO}/qemu/default-configs/*.mak ]; then
```

Comment:
@justin-he - this works, but it's fragile - if there is `>1` file matching that `*.mak`, the test will fail with an error. If you cannot specify the exact file name, it would be safer to use something like:

```sh
local kconfigs=$(ls "${GOPATH}/src/${PACKAGING_REPO}/qemu/default-configs"/*.mak)
local count=$(echo "$kconfigs"|wc -l)

local expected=1

[ "$count" -gt "$expected" ] && die "expected $expected qemu configs, found $count"
[ "$count" -eq "$expected" ] && cp ...

```

Reply:
okay, thanks
I would choose to explicit the exact name of 2 config files: 1)aarch64-softmmu.mak  2)arm-softmmu.mak. file 1) will include file 2).
Currently, we didn't have a tuned arm-softmmu.mak, but the tuning is coming soon.
Hence, I prefer:
```
if [ -e ${GOPATH}/src/${PACKAGING_REPO}/qemu/default-configs/arm-softmmu.mak ]; then
    cp -af ${GOPATH}/src/${PACKAGING_REPO}/qemu/default-configs/arm-softmmu.mak${GOPATH}/src/${QEMU_REPO}/default-configs/
fi

if [ -e ${GOPATH}/src/${PACKAGING_REPO}/qemu/default-configs/aarch64-softmmu.mak ]; then
    cp -af ${GOPATH}/src/${PACKAGING_REPO}/qemu/default-configs/aarch64-softmmu.mak ${GOPATH}/src/${QEMU_REPO}/default-configs/
fi
```
What do you think of it? Thanks

Reply:
Hi @justin-he - yes that's fine too ;)